### ISSUE TITLE
Support running from other locations

### ISSUE THREAD

**Issue by pepijndevos** [ZirconiumX/74xx-liberty#10]
Today I was experimenting with using this project with the experimental GHDL Yosys plugin: https://github.com/tgingold/ghdlsynth-beta

I ran into issues because the synth script has hardcoded paths relative to the current directory, not the scripts directory, so it only works from subfolders of this repo. So that doesn't really work when you want to use it in another project. (Like a TTL dual stack CPU :)

@daveshah1 mentioned that there is a TCL API, so it should be possible to encode the paths relative to the script file with some trickery. `help tcl` tells me it can import the yosys commands into the TCL shell, so it just needs a few lines to make the correct paths.

While I'm at it I might also add VHDL support to the Makefile, but that stuff is very experimental and requires a lot of extra setup. So maybe I'll just keep that for my own entertainment until it's a bit more stable and complete. (I plan to contribute to GHDL a bit over the summer)

**Comment by pepijndevos** [comment]
Actually I think the right thing to do might be to write a plugin in C and copy all the files into the plugin folder. That way you can just `synth_74xx` from everywhere.

**Comment by ZirconiumX** [comment]
Yeah, probably, since we're probably not going to get this upstreamed.
